src/models/shape_loader.cpp

```diff
@@ -113,7 +250,7 @@ geo::ShapePtr getHeightMapShape(const std::string& image_filename, const geo::Ve
                 std::vector<geo::Vec2i> points, line_starts;
                 findContours(image, geo::Vec2i(x, y), 0, points, line_starts, contour_map);
 
-                unsigned int num_points = points.size();
+                unsigned int num_points = (unsigned int) points.size();
```

Comment:
Normally you'd use `std::size_t` type for sizes and counts rather than an integer.

Reply:
That would be better, but not used in this file. So for consistency keeping it as it is.